### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by harryBedfordSL**
**Made styling changes including:**
- grid format for books
- Scott Logic Logo
- Animated check-boxes
- Font change

### CODE REVIEW

File: src/main/client/public/index.html [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8">
+  <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <meta name="theme-color" content="#000000">
+  <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
+  <title>React App</title>
```
Comment (d-withers) [inline src/main/client/public/index.html]: While we're styling should we change the title?

File: src/main/client/src/common/modals/DeleteBookModal.js [hunk 2]
```diff
@@ -25,13 +25,16 @@ export default class DeleteBookModal extends Component {
     return (
       <div>
         <Modal
+          id="modal"
           trigger={
-            <button
+            <Button
+              disabled={!(this.props.deleteList.length > 0)}
```
Comment (Ro4052) [inline src/main/client/src/common/modals/DeleteBookModal.js]:
```suggestion
              disabled={this.props.deleteList.length}
```

File: src/main/client/src/dashBoard/DashBoard.js [hunk 3]
```diff
@@ -6,6 +6,8 @@ import DeleteBookModal from "../common/modals/DeleteBookModal";
 import { connect } from "react-redux";
 import * as bookListActions from "./bookList/reducer";
 import { bindActionCreators } from "redux";
+import { Button } from "semantic-ui-react";
+import ScottLogicLogo from "../common/ScottLogicLogo.jpg";
```
Comment (Ro4052) [inline src/main/client/src/dashBoard/DashBoard.js]: This shouldn't have a capital letter for the variable name

File: src/main/client/src/dashBoard/addBook/AddBook.module.css [hunk 4]
```diff
@@ -0,0 +1,4 @@
+.addBook {
+    flex: 1;
+    margin: 2em;
+}
```
Comment (Ro4052) [inline src/main/client/src/dashBoard/addBook/AddBook.module.css]: Umm...

File: src/main/client/src/dashBoard/bookList/BookList.module.css [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+    flex: 10;
+    overflow-y: auto;
+}
+
+.bookList {
+    margin: 0px;
+    display: flex;
+    flex-wrap: wrap;
+    list-style: none;
+}
```
Comment (Ro4052) [inline src/main/client/src/dashBoard/bookList/BookList.module.css]: 😡

File: src/main/client/src/dashBoard/bookList/book/Book.module.css [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+input[type=checkbox] + label:active:before {
+    transform:scale(0);
+}
+
+input[type=checkbox]:checked + label:before {
+    background-color: mediumseagreen;
+    border-color: mediumseagreen;
+    color: #fff;
+}
```
Comment (Ro4052) [inline src/main/client/src/dashBoard/bookList/book/Book.module.css]: 😡😡😡

File: src/main/client/public/index.html (hunk 1 above)
Comment (harryBedfordSL) [reply src/main/client/public/index.html]: changed

File: src/main/client/src/common/modals/DeleteBookModal.js (hunk 2 above)
Comment (harryBedfordSL) [reply src/main/client/src/common/modals/DeleteBookModal.js]: changed

File: src/main/client/src/dashBoard/DashBoard.js (hunk 3 above)
Comment (harryBedfordSL) [reply src/main/client/src/dashBoard/DashBoard.js]: changed

**Comment by harryBedfordSL** [comment]
@Ro4052  and @d-withers I have made changes due to your comments, please re-review

**Pull request merged**